catalog-info.yaml

```diff
@@ -32,3 +35,35 @@ spec:
         everyone:
           access_level: READ_ONLY
         search-productivity-team: {}
+
+---
+# ------------------------------------------------------------------------------
+# docker image build and publish - manual release
+
+apiVersion: "backstage.io/v1alpha1"
+kind: "Resource"
+metadata:
+  name: "connectors-docker-build-publish"
+  description: "Docker image build and publish for Elastic crawler"
+  links:
+    - title: "Crawler Docker Build and Publish"
+      url: "https://buildkite.com/elastic/crawler-docker-build-publish"
```

Comment:
:sus: this 404's, and IDK why. I would have expected to be able to run this pipeline off of this PR branch to test it.

We may need to first merge this `catalog-info.yml` change along with an empty pipeline, and then _iterate_ on that pipeline in a subsequent PR.

Reply:
I just assumed it would be automagically created once merging this PR.
Here's a PR for that: https://github.com/elastic/crawler/pull/104

Reply:
I'd thought so too. 🤷